### DISCUSSION TITLE
SDLC Member Feedback: #1799 IAP Transport Refactor

### DISCUSSION THREAD

**Discussion by joeljfischer** [smartdevicelink/sdl_ios#1920] [General]
The SDLC review period for this branch starts now and continues through March 12, 2021. Please place any review feedback about the branch to fix #1799 (bugfix/7.1_transport_fixes) here so issues can be addressed as soon as possible. 

Updates included in this branch change the way that the SDL iOS library connects over iAP to modules; refactoring a lot of the existing code in order to try and make it less error-prone. Due to it being a fairly substantial change to the iAP connection code it is very important that OEMs and anyone developing the modules test these PRs to ensure there is no degradation to the stability of the modules in the next release of the iOS library (7.1.0).

**Comment by mrapitis** [comment]
We have integrated the 7.1 transport fixes branch into multiple sample apps and thus far connectivity and testing has been positive. We have tested multiple scenarios including iAP over BT and USB as well as transport switching between the two. Thus far no regressions or degradation to the stability have been observed.

**Reply by mrapitis** [reply]
As a side note, we are still seeing issue https://github.com/smartdevicelink/sdl_ios/issues/1783 which is also related to app connectivity. Would it be possible to plan for including the suggested comment linked below in a future release and provide a review period similar to what has been done for the iAP refactor?

https://github.com/smartdevicelink/sdl_ios/issues/1783#issuecomment-721088814

**Reply by jordynmackool** [reply]
@mrapitis Thank you for your feedback. If Ford wishes to prioritize issue https://github.com/smartdevicelink/sdl_ios/issues/1783 for a future release, please request to add it to the issue triage list during the Steering Committee meeting.

**Comment by jordynmackool** [comment]
The Steering Committee review period for branch `bugfix/7.1_transport_fixes` is now complete. Branch `bugfix/7.1_transport_fixes` will be included in the April 14, 2021 release.
